config/routes.rb

```diff
@@ -244,10 +244,13 @@ def patch(*)
       end
 
       scope "/customize", constraints: AdminConstraint.new do
+        resources :site_settings, only: %i[index]
         resources :form_templates, constraints: AdminConstraint.new, path: "/form-templates" do
           collection { get "preview" => "form_templates#preview" }
         end
 
+        get "font" => "site_settings#index"
```

Comment:
Route should be within the `:config` namespace, see https://github.com/discourse/discourse/pull/30528 for an example